core/src/contrib/components/actions/WASDMovement.ts

```diff
@@ -19,6 +19,12 @@ export const WASDMovementPhysics: ActionMap<WASDMovementActions> = {
 
 const move = (entity: Entity<Position>, animation: string | undefined, velocity: { x: number, y: number }) => {
   const { position, renderable } = entity.components;
+
+  if (currentJoystickPosition.power) {
+    velocity.x *= currentJoystickPosition.power;
+    velocity.y *= currentJoystickPosition.power;
+  }
```

Comment:
`currentJoystickPosition` is tech debt, need to refactor joystick input -> controller